packages/fetchai/connections/local/connection.py

```diff
@@ -388,27 +388,17 @@ class OEFLocalConnection(Connection):
     It is useful for local testing.
     """
 
-    def __init__(
-        self,
-        address: Address,
-        local_node: LocalNode,
-        connection_id: str = "local",
-        restricted_to_protocols: Optional[Set[str]] = None,
-        excluded_protocols: Optional[Set[str]] = None,
-    ):
+    def __init__(self, address: Address, local_node: LocalNode, *args, **kwargs):
         """
         Initialize a OEF proxy for a local OEF Node (that is, :class:`~oef.proxy.OEFLocalProxy.LocalNode`.
 
         :param address: the address used in the protocols.
         :param local_node: the Local OEF Node object. This reference must be the same across the agents of interest.
+        :param connection_id: the connection id.
         :param restricted_to_protocols: the only supported protocols for this connection.
         :param excluded_protocols: the excluded protocols for this connection.
         """
-        super().__init__(
-            connection_id=connection_id,
-            restricted_to_protocols=restricted_to_protocols,
-            excluded_protocols=excluded_protocols,
-        )
+        super().__init__(*args, **kwargs)
```

Comment:
this is an example of what I wrote before. `connection_id`, `restricted_to_protocold` and `excluded_protocols` are populated by the framework, since such information is retrieved from the configuration file.